shard.yml

```diff
@@ -28,6 +28,9 @@ dependencies:
   place_calendar:
     github: PlaceOS/calendar
 
+  office365:
```

Comment:
this is included by `place_calendar` so can be removed
pull the latest version using `shards update` instead